enigma-principal/app/src/boot_network/principal_manager.rs

```diff
@@ -408,6 +409,7 @@ mod test {
         env::set_var("MAX_EPOCHS","10");
         env::set_var("SPID", "B0335FD3BC1CCA8F804EB98A6420592D");
         env::set_var("ATTESTATION_SERVICE_URL", "https://sgx.enigma.co/api");
+        env::set_var("ATTESTATION_RETRIES", "11");
```

Comment:
Just out of curiosity, why the value is different from the value in core?

Reply:
I wanted to make sure that it's getting the value from the environment variable and not as the default value when initializing the KM without anything.